metadata_annotation.ipynb

```diff
@@ -0,0 +1,2307 @@
+{
```

Comment:
<p>Why not use <a href="https://scanpy.readthedocs.io/en/stable/generated/scanpy.pp.subsample.html" target="_blank">https://scanpy.readthedocs.io/en/stable/generated/scanpy.pp.subsample.html</a> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>